### PR TITLE
Add arrow button navigation

### PR DESCRIPTION
**Pull request by MahmoudHamdy02**

### CODE REVIEW

**Review by KKoukiou — CHANGES_REQUESTED** [review]
I have some concern regarding the implemetation of this .
SO you consider 0 index the /home/oneuser directory right? This is slightly confusing because you want to be able to reach also / so that would give you negative indexes for these directories. 

SO I believe it would be better implemeted if using full paths only for the state, not the indexes.

Also when performing the navigation the breadcrumb should also change.

Lastly the back button should be disabled when there is nowhere back to go to (being at /)


I am not sure what the forward button is meant for in the mockups, I need to recheck this.

**Comment by MahmoudHamdy02** [comment]
I've adjusted most of the points except for starting the navigator in the user home directory, since it kept giving me infinite loops and memory leaks :/
I'll look into it again when the `cockpit.channel` is added

File: src/app.jsx [hunk 1]
```diff
@@ -79,14 +97,18 @@ const NavigatorBreadcrumbs = ({ path, setPath }) => {
                     </Button>
                 </FlexItem>
                 <FlexItem>
-                    <Breadcrumb>
-                        <BreadcrumbItem to="#" onClick={() => setPath([])}>{_("home")}</BreadcrumbItem>
-                        {path.map((dir, i) => {
+                    <Flex>
```
Comment (KKoukiou) [inline src/app.jsx:100]: It's easy to fix the spacing,
`<Flex spaceItems={{ default: "spaceItemsXs" }}>` will decrease it.

**Review by KKoukiou — CHANGES_REQUESTED** [review]

**Comment by MahmoudHamdy02** [comment]
Added the requested changes

**Review by KKoukiou — APPROVED** [review]

**Pull request merged**